File: python-venv/equal_sides_of_an_array/kata_function/equal_sides_of_an_array.py
def find_even_index(array):

    for i in range(0, len(array), 1):
        left = 0
        right = 0
        for j in range(0, len(array), 1):
            if(i > j):
                left += array[j]
            elif(j > i):
                right += array[j]
        if left == right:
            return i

    return -1

#can you loop & slice either side of i, then count the slice & evaluate?

#https://www.codewars.com/kata/5679aa472b8f57fb8c000047/train/python

File: equal-sides-of-an-array/equalSidesOfAnArray.js
const findEvenIndex = (array) => {

    for(let i = 0; i < array.length; i++) {
        let left = 0;
        let right = 0;
        for(let j = 0; j < array.length; j++) {
            if(i > j) left += array[j];
            else if (j > i) right += array[j];
        };

    if (left === right) return i;
    };

    return -1;
};

module.exports = { findEvenIndex };


//https://www.codewars.com/kata/5679aa472b8f57fb8c000047/train/javascript
